opentracing/src/main/java/opentracing/propagation/Extractor.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import opentracing.Span;
+
+/**
+ */
+public interface Extractor<T> {
```

Comment:
this (and `Injector<T>`) don't need to be part of `opentracing-java` now that `Tracer` has `inject(...)` and `join[Trace](...)`.